artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/Create.java

```diff
@@ -283,6 +283,12 @@
    @Option(name = "--staticCluster", description = "Cluster node connectors list, separated by comma: Example \"tcp://server:61616,tcp://server2:61616,tcp://server3:61616\"")
    String staticNode;
 
+   @Option(name = "--support-advisory", description = "Set supportAdvisory=true for the main 'artemis' acceptor on OPENWIRE protocol (Default: false)")
+   boolean supportAdvisory = false;
+
+   @Option(name = "--suppress-internal-management-objects", description = "Set suppressInternalManagementObjects=true for the main 'artemis' acceptor on OPENWIRE protocol (Default: false)")
+   boolean suppressInternalManagementObjects = false;
```

Comment:
Same, here,

description = "If --supress-internal-management-objects is used the generated configuration will allow Internal Management Objects for the OpenWire protocol"



if you could find a better description for these objects it would be even better.

Reply:
shouldn't these by the same attribute used?

you create a single CLI attribute such as --openwire-advisory


and use the same attribute on both places?

Reply:
By default people won't even notice, but I have left them to be separated as the acceptor params are (can be tuned singularly)